delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -43,10 +44,16 @@ class ResourcesRoutesSpec extends BaseRouteSpec with IOFromMap with CatsIOValues
   private val uuid                  = UUID.randomUUID()
   implicit private val uuidF: UUIDF = UUIDF.fixed(uuid)
 
-  implicit private val caller: Caller =
+  private val reader = alice
+  private val writer = bob
+
+  implicit private val callerAlice: Caller =
     Caller(alice, Set(alice, Anonymous, Authenticated(realm), Group("group", realm)))
+  implicit private val callerBob: Caller   =
+    Caller(bob, Set(bob, Anonymous, Authenticated(realm), Group("group", realm)))
 
-  private val asAlice = addCredentials(OAuth2BearerToken("alice"))
+  private val asReader = addCredentials(OAuth2BearerToken("alice"))
+  private val asWriter = addCredentials(OAuth2BearerToken("bob"))
```

Comment:
Introducing aliases for alice and bob so that it's clear what the user can do

Reply:
Great change 👍